var/spack/repos/builtin/packages/pixman/package.py

```diff
@@ -31,6 +31,11 @@ class Pixman(AutotoolsPackage):
     depends_on("bison@3:", type="build")
     depends_on("libpng")
 
+    variant("shared", default=True, description="Build shared library")
+    variant("pic", default=False, description="Enable position-independent code")
+
+    conflicts("+shared ~pic")
```

Comment:
@tldahlgren / @AlexanderRichert-NOAA: should've been `conflicts("~shared ~pic")`. Static lib should have -fPIC.

Right now the concretizer has to choose between `+shared +pic` or `~shared ~pic`, they're both optimal solutions. The latter makes `cairo` fail to build on develop.

I don't get the point of making `pic` a variant. I've removed it in #45272.